frameworks/PHP/symfony/composer.json

```diff
@@ -1,72 +1,44 @@
 {
-	"name" : "symfony/framework-standard-edition",
-	"license" : "MIT",
 	"type" : "project",
-	"description" : "The \"Symfony Standard Edition\" distribution",
-	"autoload" : {
-		"psr-4" : {
-			"" : "src/"
-		},
-		"classmap" : [
-			"app/AppKernel.php",
-			"app/AppCache.php"
-		]
-	},
-	"autoload-dev" : {
-		"psr-4" : {
-			"Tests\\" : "tests/"
-		},
-		"files" : [
-			"vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
-		]
-	},
+	"license" : "proprietary",
 	"require" : {
-		"php" : ">=5.5.9",
-		"doctrine/doctrine-bundle" : "^1.6",
-		"doctrine/orm" : "^2.5",
-		"incenteev/composer-parameter-handler" : "^2.0",
-		"sensio/distribution-bundle" : "^5.0.19",
-		"sensio/framework-extra-bundle" : "^3.0.2",
-		"symfony/monolog-bundle" : "^3.1.0",
-		"symfony/polyfill-apcu" : "^1.0",
-		"symfony/swiftmailer-bundle" : "^2.3.10",
-		"symfony/symfony" : "^3.3",
-		"twig/twig" : "^1.0||^2.0"
+		"php": "^7.0.8",
+		"doctrine/doctrine-bundle": "^1.6",
+		"doctrine/orm": "^2.5",
+		"symfony/console": "^3.4",
+		"symfony/flex": "^1.0",
+		"symfony/framework-bundle": "^3.4",
+		"symfony/lts": "^3",
+		"symfony/twig-bundle": "^3.4",
+		"symfony/yaml": "^3.4",
```

Comment:
for now this is 3.4
can't wait for a 4.0 upgrade :)

Reply:
I didn't noticed it was 3.4. I thought Symfony 4 was always used when starting a new flex project (composer create-project symfony/skeleton).